### DISCUSSION TITLE
How To Move Full Node From Windows To Linux

### DISCUSSION THREAD

**Discussion by subz390** [Chia-Network/chia-blockchain#5025] [Q&A]
Is there a guide somewhere on how to move a full node from Windows to Linux?
Or is it so simple you don't need a guide - like;
1. install Chia fresh on the new system
2. enter your mnemonic
3. bam your wallet magically appears with the correct balance
4. everything just works, happy days...

**Comment by cccat6** [comment]
You are right. But it will sync for a long time. If you are looking for a real "move". Just copy the DB files from your original computer to save your time on syncing.
Remember, the only thing that recognizes you on a blockchain is your primary key.

**Reply by subz390** [reply]
Ok thanks for the info on syncing.

And to double check - do I have to manually copy the wallet keypair or does entering the mnemonic recreate those at the new install location?  ie these files `C:\Users\<user>\.chia\mainnet\config\ssl\wallet\(private|public)_wallet.(key|crt)` files.
or for that matter all the files in the `ssl` folder, or does Linux use different file types for certificates etc.

**Reply by cccat6** [reply]
C:\Users\<user>\.chia\mainnet
Then you see there are many folders. Only "db" and "wallet/db" are necessary to copy.
Remember, in your new machine. Input the key at first, when it goes to the main page. CLOSE IT.
Then you can copy the files. Because when the program is running, it will lock the files.

**Reply by MarcusL11** [reply]
HI, is this still true for today, as its the latest based on current version?
